### ISSUE TITLE
The issue of secondary spectrum

### ISSUE THREAD

**Issue by morestart** [levitsky/pyteomics#115]
If the mzML file contains the "spectrumList" field, data can be parsed, but if the field is "chromatogramList," the data cannot be parsed.

**Comment by mobiusklein** [comment]
It's not clear how you're trying to use the library. If you can share a snippet of code, that would help understand what's not working as expected.

Guessing at what you might be doing, by default, the `MzML` class will iterate over `spectrum` elements. If you want to access chromatograms, you need to explicitly ask for it by calling the `iterfind` method:
```pyteomics
from pyteomics.mzml import MzML

reader = MzML("path/to/file.mzML")

for chrom in reader.iterfind("chromatogram"):
    print(chrom)
```

**Comment by morestart** [comment]
You can find this file from https://github.com/morestart/TAFA-LAMS/blob/master/assets/WD%20tip-YP-SD116-CH3CH2OH-01NH4OH-QX-EPA-MRM-1.mzML

Your code is not working, i got this error:
`TypeError: only size-1 arrays can be converted to Python scalars`

**Comment by morestart** [comment]
Sorry~Now the repository is public.

**Comment by morestart** [comment]
any solutions?

**Comment by levitsky** [comment]
I suppose we can just add an exception handling clause around this statement. The latest commit in `master` implements this.

**Comment by morestart** [comment]
> I suppose we can just add an exception handling clause around this statement. The latest commit in implements this.`master`

Yes, I changed this code to your commit. It works. Thanks for your work!

**Comment by nkitagawa-venn** [comment]
> I suppose we can just add an exception handling clause around this statement. The latest commit in `master` implements this.

@levitsky I came across this issue today as well.  Another way to fix this might be to change `info[k] = int(info[k])` into `info[k] = info[k].astype(int)`, which would more directly get across the transformation you are trying to apply.  What do you think?

**Comment by mobiusklein** [comment]
The majority of values in `info` are not `numpy` objects, so `astype` isn't available for them. This particular case is being caused by the expectation that `ms level` is referring to the CV term `MS:1000511`, which is a scalar value, but it is being used as a non-standard array name (if the custom array were named `ms level array`, this wouldn't be an issue).

Binary data arrays, even non-standard ones, have their types explicitly given by a CV term already, so they don't need to be coerced the way other `cvParam` values and attribute values are.

**Comment by nkitagawa-venn** [comment]
> The majority of values in `info` are not `numpy` objects, so `astype` isn't available for them. This particular case is being caused by the expectation that `ms level` is referring to the CV term `MS:1000511`, which is a scalar value, but it is being used as a non-standard array name (if the custom array were named `ms level array`, this wouldn't be an issue).
> 
> Binary data arrays, even non-standard ones, have their types explicitly given by a CV term already, so they don't need to be coerced the way other `cvParam` values and attribute values are.

Thank you @mobiusklein for the explanation!
